server/index.js

```diff
@@ -1,18 +0,0 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const ac = require(__dirname+'/controllers/animalcontroller')
-const cors = require('cors')
-const app = express()
-
-app.use( bodyParser.json())
-app.use(cors())
-
-const apiBaseUrl = "/api/animals"
-
-app.post(apiBaseUrl, ac.create)
-app.get(apiBaseUrl, ac.read)
-app.put(`${apiBaseUrl}/:desc&:id`, ac.update)
```

Comment:
interesting choice to put the new description in the url - since the description could be long I would probably include this with the body but that is a personal preference thing.